app/assets/stylesheets/admin/loans/_questionnaires.scss

```diff
@@ -136,6 +137,32 @@ section.questionnaires {
       margin: 0;
     }
   }
+
+
+  .form-element.actions {
+    // When position is fixed, the entire webpage/viewport is the parent element
+    position: fixed;
+    bottom: 0;
+    right: $content-wrapper-padding;
+    display: block;
+    margin: 0 0 $content-wrapper-padding;
+    padding: 1em;
+    // Use built-in Sass function and color fallback
+    background: $gray2;
+    background: rgba($gray2, 0.5);
+    text-align: right;
+
+    $questionnaire-actions-offset: 2*#{$content-wrapper-padding};
+    width: calc(100% - #{$questionnaire-actions-offset});
```

Comment:
Inside calc, interpolation is needed. But I was able to remove it for `2 * $content-wrapper-padding`.